packages/ui-components/src/components/Matrix/Cell.tsx

```diff
@@ -0,0 +1,14 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { TableCell } from '@material-ui/core';
+import styled from 'styled-components';
+import { getFullHex } from './utils';
+
+export const Cell = styled(TableCell)`
+  border-width: 0 1px 1px;
```

Comment:
Possibly a silly question, but what’s the rationale behind omitting the top border?

Reply:
Not a silly question :) Because we add the bottom border, if we also add a top border it will be double thickness, which we don't want in this case

Reply:
Huh, are we not using `border-collapse: collapse`? This might be another thing we want to consider resetting with `MuiCssBaseline` 🤔

But in this case, I vaguely remember from working in DataTrak that MUI’s `stickyHeader` _explicitly_ uses `border-collapse: separate`, so will defer to what you’ve already got here

Reply:
Yes that's right - setting `border-collaps: separate` has some knock on effects for the sticky header once you're scrolling (it removes the border altogether, which isn't desirable) and I think that's why MUI have set this